### DISCUSSION TITLE
Filesystem-based catalog

### DISCUSSION THREAD

**Discussion by SergeiPatiakin** [apache/iceberg-rust#695] [Q&A] [answered]
Does iceberg-rust support the File System Table catalog? https://iceberg.apache.org/spec/#file-system-tables

Would we like to support it? Do we plan to support it?

**Comment by Xuanwo** [comment, marked as answer]
Hi, the community has decided to deprecate this catalog. There are no plans to support it. However, I am personally open to this if you have specific use cases.

By the way, @jackye1995 might also be interested.
